pkg/controllers/managementuser/resourcequota/resource_quota_sync.go

```diff
@@ -450,9 +450,11 @@ func (c *SyncController) getResourceLimitToUpdate(ns *corev1.Namespace) (*corev1
 		return convertPodResourceLimitToLimitRangeSpec(updatedLimit)
 	} else if nsLimit != nil {
 		return convertPodResourceLimitToLimitRangeSpec(nsLimit)
+	} else if projectLimit != nil {
+		return convertPodResourceLimitToLimitRangeSpec(projectLimit)
+	} else {
+		return nil, nil
 	}
```

Comment:
Nit: Since you technically return right after this, you don't really need the `else{}` clause. The if block will just fall through to the return case in the situation where we don't have a combo limit, project limit, or namespace limit.

That being said, this is fine since it makes that behavior more explicit, so I don't think it really matters which solution you chose. Up to you - feel free to resolve after you decide what you want to do.

Reply:
This is just a matter of style, two `else if` clauses look dangling without the concluding `else`, which is indeed a catch-all clause here, since it returns `nil, nil`.